distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -91,41 +98,27 @@ new_builder (capacity=10) = Builder.new capacity
    A vector allows to store an arbitrary number of elements in linear memory. It
    is the recommended data structure for most applications.
 from_polyglot_array : Any -> Vector Any
-from_polyglot_array arr = Vector_Data (Proxy_Polyglot_Array.Proxy_Polyglot_Array_Data arr)
-
-## The basic, immutable, vector type.
-type Vector a
-
-    ## ADVANCED
+from_polyglot_array array = @Builtin_Method "Vector.from_polyglot_array"
 
-       The basic, immutable, vector type.
-
-       Arguments:
-       - storage: The underlying storage.
+##  The basic, immutable, vector type.
+    A vector allows to store an arbitrary number of elements, in linear memory.
+    It is the recommended data structure for most applications.
 
-       A vector allows to store an arbitrary number of elements, in linear memory.
-       It is the recommended data structure for most applications.
+    > Example
+      A vector containing the elements `1`, `2`, and `3`, in this order is:
+          [1, 2, 3]
 
-       > Example
-         A vector containing the elements `1`, `2`, and `3`, in this order is:
-             [1, 2, 3]
+    > Example
+      A vector containing 50 elements, each being the number `42`, can be
+      created by:
+          Vector.fill length=50 item=42
 
-       > Example
-         A vector containing 50 elements, each being the number `42`, can be
-         created by:
-             Vector.fill length=50 item=42
-    Vector_Data storage
+@Builtin_Type
+type Vector a
```

Comment:
I'm not really convinced we should leave the `a` field here as we don't do it for other types either atm

Reply:
What are the other types? `Date`, `Time`? Well, they are not parametrized from definition.